examples/index.md

```diff
@@ -178,64 +180,35 @@ Content-Type: application/vnd.api+json
 {
   "errors": [
     {
+      "status": "422",
+      "source": { "pointer": "/data/attributes/first-name" },
       "title":  "Invalid Attribute",
-      "source": { "pointer": "data/attributes/first-name" },
       "detail": "First name must contain at least three characters."
     }
   ]
 }
 ```
 
-Notice that the attribute `first-name` is dasherized.  Dasherized attributes are preferred
-over underscored.
-
-Notice also that the `detail` is specific to this occurrence of the problem,
-whereas the `title` is more generic.
-
-Notice that the `source` points to the associated attribute with the error.
-
-Response with one error object, with relationship error:
-
-```http
-HTTP/1.1 422 Unprocessable Entity
-Content-Type: application/vnd.api+json
-
-{
-  "errors": [
-    {
-      "status": "422",
-      "title":  "Missing Association",
-      "source": { "pointer": "data/relationships/author" },
-      "detail": "Post author cannot be blank."
-    }
-  ]
-}
-```
-
-Notice that the `source` points to the associated relationship with the error.
+Every member in an error object is optional, but all help the client
+by providing extra details.
 
-Notice also that the error object includes a `status` code. Though optional,
-it can be useful when the code processing the JSON isn't directly aware of
-the HTTP response or even if the JSON is coming via non-HTTP protocols.
+The `source` member is used to [indicate](https://tools.ietf.org/html/rfc6901)
+which part of the request document caused the error.
 
-Response with one error, using a `code`:
+The `title` and `detail` members are similar, but `detail` is specific
+to this occurrence of the problem, whereas `title` is more generic.
 
-```http
-HTTP/1.1 422 Unprocessable Entity
-Content-Type: application/vnd.api+json
+The `status` member represents the HTTP status code associated with the problem.
+It's very helpful when multiple errors are returned at once (see below), as the
+HTTP response itself can only have one status code. However, it can also be
+useful for single errors, to save clients the trouble of consulting the HTTP
+headers, or for using JSON API over non-HTTP protocols, which may be officially
+supported in the near future.
 
-{
-  "errors": [
```

Comment:
I got rid of this example because it just doesn't make sense...it's a client side error, while these examples are for errors returned by the server.